client/components/data/query-domains-suggestions/index.jsx

```diff
@@ -21,7 +21,7 @@ function getQueryObject( props ) {
 		query: props.query,
 		recommendation_context: props.recommendationContext,
 		vendor: props.vendor,
-		vertical: props.surveyVertical,
+		vertical: props.verticalId || props.surveyVertical,
```

Comment:
It's probably a good idea to make a helper function you pass `props` to, and it returns the vertical id or string, so we can easily change it in the future.